src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -143,6 +144,7 @@ bool ov::pass::MOCTransformations::run_on_model(const std::shared_ptr<ov::Model>
     // In particular, if zero dim tensor is consumed in body of MultiSubGraphOp
     // RemoveConcatZeroDimInput and RemoveMultiSubGraphOpDanglingParamsResults should be called together.
     using namespace ov::pass;
+    REGISTER_PASS(manager, MarkCompressedFloatConstants)
```

Comment:
Now this transformation is executed twice for frontends except PyTorch.
What value does this movement bring?

Reply:
@itikhono, @slyalin, any plans to move MOC to read_model?